nix/sources.nix

```diff
@@ -0,0 +1,148 @@
+# This file has been generated by Niv.
```

Comment:
I guess this file should be re-generated whenever we change `nix/sources.json` ?

I will try to commit it separately in its own commit before merging.

Reply:
@noteed no, this is just something niv generates that is used to parse sources.json. It is more like an interface between pinned sources in sources.json & nix.